### PR TITLE
window_title: Add format

### PR DESCRIPTION
**Pull request by lasers**
This adds format: string to print (default "{title}")

Ongoing efforts to `FORMAT ALL THE THINGS!` (meme). https://github.com/ultrabug/py3status/issues/98

### CODE REVIEW

File: py3status/modules/window_title.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 Configuration parameters:
     cache_timeout: How often we refresh this module in seconds (default 0.5)
+    format: string to print (default '{title}')
```
Comment (tobes) [inline py3status/modules/window_title.py]: changing `string to print` would be nice :)

File: py3status/modules/window_title.py [hunk 2]
```diff
@@ -41,27 +42,27 @@ class Py3status:
...
 
     def window_title(self):
         window = find_focused(i3.get_tree())
 
         transformed = False
-        if window and 'name' in window and window['name'] != self.text:
-            self.text = (len(window['name']) > self.max_width and
+        if window and 'name' in window and window['name'] != self.title:
+            self.title = (len(window['name']) > self.max_width and
                          "..." + window['name'][-(self.max_width - 3):] or
```
Comment (tobes) [inline py3status/modules/window_title.py]: I think this may also have the unicode concat issue, maybe I'm paranoid

Comment (lasers) [inline py3status/modules/window_title.py]: Paranoid or not, what do you want to change it to? I will. :-)

Comment (tobes) [inline py3status/modules/window_title.py]: `u"...{}".format(window['name'][-(self.max_width - 3):]) or`

the whole bit of code here makes me a little sick in my mouth

**Comment by ultrabug** [comment]
Clean, thanks !

**Pull request merged**